models/yang/lint_ignore.ocstyle

```diff
@@ -0,0 +1 @@
+openconfig-file-mgmt-private.yang
```

Comment:
Linter is rejecting this lint_ignore configuration; hence the build error. I think it accepts only the yang module names here and not yang file names. Change to "openconfig-file-mgmt-private".

Reply:
This was part of it, but the doctree.py pyang plugin was not up to date with the private repo and was also causing an issue.